libfreerdp/core/rdp.c

```diff
@@ -604,7 +604,7 @@ BOOL rdp_send_data_pdu(rdpRdp* rdp, wStream* s, BYTE type, UINT16 channel_id)
 	Stream_SetPosition(s, length);
 	Stream_SealLength(s);
 
-	WLog_DBG(TAG, "%s: sending data (type=0x%x size=%d channelId)", __FUNCTION__, type, Stream_Length(s), channel_id);
+	WLog_DBG(TAG, "%s: sending data (type=0x%x size=%d channelId=%d)", __FUNCTION__, type, Stream_Length(s), channel_id);
```

Comment:
Looks like there are a few issues here (not your fault, but none the less):
* ```Stream_GetPosition``` returns ```sizt_t```
  * Type of ```length``` is ```UINT16```
  * Type of ```sec_hold``` is ```int```
* Debug format string uses wrong format specifiers

Reply:
ok will change that